mobile/lib/features/trade/channel_configuration.dart

```diff
@@ -48,7 +50,7 @@ channelConfiguration({
                 },
                 child: SingleChildScrollView(
                   child: SizedBox(
-                    height: 450,
+                    height: 500,
```

Comment:
I've had to bump this number even further because otherwise the unit tests fail 🙈

Reply:
Noo 🫣 I worked so long to get the tests pass with the correct size 😅

Now it looks weird again 🙈